include/common/DeviceMatrix.h

```diff
@@ -448,15 +507,16 @@ template <typename T> class DeviceMatrices {
         if (src.n() != dest.m() || src.m() != dest.n()) throw std::invalid_argument("Cannot transpose.");
```

Comment:
Having the PackedTranspose working on DeviceMatrixViews but being part of the DeviceMatrix class does not really make sense

Reply:
postponing the design, as design was by CB :)